README.md

```diff
@@ -1,12 +1,48 @@
+
 [![Join the chat at https://gitter.im/fetch-amul-topicals/Lobby](https://badges.gitter.im/fetch-amul-topicals/Lobby.svg)](https://gitter.im/fetch-amul-topicals/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-A Python script that fetches details all ‘topicals’ across all years from the Amul website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures by year into your local system
+##Table of Contents
+* Introduction
+ * Steps
+ * Requirements
+* Installation Guide
+* Screenshot
+* License
+
+## Introduction
+It uses a Python script that fetches details (here ‘topicals’) across all years from the Amul's website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures chronologically into your local system.
+
+### Steps
+1.Import necessary libraries <br>
+2.Go to the website <br>
+3.Find the sequence in the URL of the images. Usually they are in consecutive numbers <br>
+4.Use the method of webscraping <br>
+5.Use try-catch blocks for exception might arise in between <br>
 
-## Amul Website
+### Requirements
```

Comment:
Requirements aren't correct, we don't use Ruby for example. Please correct this and refer to the code for the exact libraries used

Reply:
- Chuck those libraries that are already there when one installs Python. (like os, sys).
- What is 'import'?